src/app/members/components/MemberActionRender.tsx

```diff
@@ -1,13 +1,13 @@
-import React from 'react';
-import ActionsMenu from '@/common/ActionMenu/ActionsMenu';
+import React from "react";
+import ActionsMenu from "@/common/ActionMenu/ActionsMenu";
 import { Info, Trash2, Edit, Send } from "lucide-react";
-import { MemberRole } from '@/lib/constants';
+import { MemberRole } from "@/lib/constants";
 
 interface MemberInfo {
   id: number;
-  name: string;
-  phone: string;
-  role: string;
+  name?: string;
+  phone?: string;
+  role?: string;
```

Comment:
@ghulamhafeez why it need to be optional?

Reply:
For the time being, we’re using a workaround. I’ve also mentioned this in the PR, and we’ve created a separate ticket to address it